cloudflare_worker/README.md

```diff
@@ -48,7 +48,7 @@ limitations under the License.
    1. The terminal will interactively ask for the value of the secret.
```

Comment:
Add a step above line 24 to `git clone` this repo and cd into cloudflare_worker.

Reply:
Done